iron-list.html

```diff
@@ -413,7 +413,9 @@
     behaviors: [
       Polymer.Templatizer,
       Polymer.IronResizableBehavior,
-      Polymer.IronScrollTargetBehavior
+      Polymer.IronScrollTargetBehavior,
+      /** Polymer.OptionalMutableDataBehavior is only available with Polymer 2.0. */
+      Polymer.OptionalMutableDataBehavior || {}
```

Comment:
This might break the static analysis. Consider moving it to the top. e.g.
```js
if (!Polymer.OptionalMutableDataBehavior) {
  Polymer.OptionalMutableDataBehavior = {};
}
```